src/services/api/index.ts

```diff
@@ -106,8 +106,9 @@ const parseNumber = (
 };
 
 function getOwner() {
-  const address = localStorage.getItem(LOCAL_STORAGE_ADDRESS) || ZERO_ADDRESS;
-  return Address.parse(address as string)
+  const address = localStorage.getItem(LOCAL_STORAGE_ADDRESS);
```

Comment:
@denis-orbs fyi. I see that things *do work* when I throw here, but not familiar enough with the code to know whether it's a good idea. consider refactoring to allow `getOwner()` to return `null`